sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_generated/models.py

```diff
@@ -3,11 +3,6 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
-#
-# Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is
-# regenerated.
 # --------------------------------------------------------------------------
-
-VERSION = "7.1-preview"
-
+from .v7_0.models import *
+from .v7_1.models import *
```

Comment:
Did autorest generate this? Seems strange to import the models from `v7_0` then replace them with models from `v7_1`.

Reply:
the point is to import all of the models across API versions. I'm not super familiar with api versioning, but I do see how the 7.1 models of the same name as the 7.0 models could be overwriting them. When importing models, we also allow users to get the models off of the generated multiapi client, ie keyvault_client.models(), which has an optional `api_version` parameter. 

Bringing in @lmazuel to ask whether this is behavior we want to consider changing on the autorest side

Reply:
> the point is to import all of the models across API versions.

It leaves out 2016-10-01 though. And I find nothing importing from this module, so it seems its purpose is to burn some cycles ensuring these two modules exist? This code works even if they're empty.

Reply:
Good point. I'm going to open an issue to look into this more on the autorest side